docs/connectors/io-crd-config/source-crd-config.md

```diff
@@ -16,7 +16,6 @@ This table lists source configurations.
 | `tenant` | The tenant of a source connector. |
 | `Replicas`| The number of Pulsar instances that you want to run this source connector. |
```

Comment:
| `Replicas`| The number of instances that you want to run this source connector. By default, the `Replicas` is set to `1`. |